OasisDEX/oasis-borrow
# Multiply -> Borrow transition.

# [Extend ManageMultiply Vault pipeline to implement Multiply -> Borrow transition flow](https://app.shortcut.com/oazo-apps/story/2954/extend-managemultiply-vault-pipeline-to-implement-multiply-borrow-transition-flow)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  
- Can transition from multiply vault to borrow vault, by going to the "Borrow" tab.
  
## How to test 🧪
  <Please explain how to test your changes>

- On a multiply vault go to "Borrow" and make the transition.
- Check the code.
- The borrow to multiply transition might need to be tested as well because there's some refactoring.
